 website2/index.html

```diff
@@ -0,0 +1,148 @@
+<html>
+
+<head>
+	<title>
+	</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+	<script type="text/javascript" src="validations.js"></script>
+</head>
+
+<body  class="parent">
+	<header>
+		<div class="header">
+			<div class="header-links ">
+				<ul class="link-holder">
+					<li class="logo">
+						<div>
+							<img src="appdirect_logo.png" width="30px" height="30px">
```

Comment:
use a tag instead of img tag and set its backgroud as logo url.
no inline styling in html